automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/AsyncImportSpec.scala

```diff
@@ -228,42 +238,42 @@ class AsyncImportSpec
 
     val importJobId: String = importJobIdValues.head match {
       case js: JsString => js.value
-      case x => fail("got an invalid jobId: " + x.toString ())
+      case x => fail("got an invalid jobId: " + x.toString())
     }
 
     logger.info(s"$projectName/$workspaceName has import job $importJobId for file $importFilePath")
 
     // poll for completion as owner
     withClue(s"import job $importJobId failed its eventually assertions on job status: ") {
-      eventually(timeout = Timeout(scaled(Span(10, Minutes))), interval = Interval(scaled(Span(5, Seconds)))) {
+      eventuallyWithEscape(Timeout(scaled(Span(10, Minutes))), Interval(scaled(Span(5, Seconds))), {
         val requestId = scala.util.Random.alphanumeric.take(8).mkString // just to assist with logging
-        logger.info(s"[$requestId] About to check status for import job $importJobId. Elapsed: ${humanReadableMillis(System.currentTimeMillis()-startTime)}")
+        logger.info(s"[$requestId] About to check status for import job $importJobId. Elapsed: ${humanReadableMillis(System.currentTimeMillis() - startTime)}")
         val resp: HttpResponse = Orchestration.getRequest(s"${workspaceUrl(projectName, workspaceName)}/importJob/$importJobId")
         val respStatus = resp.status
-        logger.info(s"[$requestId] HTTP response status for import job $importJobId status request is [${respStatus.intValue()}]. Elapsed: ${humanReadableMillis(System.currentTimeMillis()-startTime)}")
+        logger.info(s"[$requestId] HTTP response status for import job $importJobId status request is [${respStatus.intValue()}]. Elapsed: ${humanReadableMillis(System.currentTimeMillis() - startTime)}")
         respStatus shouldBe StatusCodes.OK
         val importStatus = blockForStringBody(resp).parseJson.asJsObject.fields.get("status").value
         logger.info(s"[$requestId] Import Service job status for import job $importJobId is [$importStatus]")
-        importStatus shouldBe JsString ("Done")
-      }
+        assertWithEscape(importStatus shouldBe JsString("Done"), importStatus should not equal JsString("Error"), "Import error occurred, failing test")
+      })
```

Comment:
now that the signatures for `eventuallyWithEscape` mirror those of `eventually`, please revert the changes to parens here - move the close-paren from line 258 to line 248

Reply:
Whoops! Forgot that part.  Thanks!